ramls/inventory.raml

```diff
@@ -84,6 +84,28 @@ resourceTypes:
               application/json:
                 example: !include examples/instance_context_get.json
                 schema: context
+  /instancesCollection:
+     post:
+        description: "Create collection of instances"
+        body:
+          application/json:
+            type: instances
```

Comment:
The implementation expects that the instances are provided as a raw array, without any surrounding JSON, that does not match this definition, where the request is an object with an `instances` array property and a `totalRecords` property.

Reply:
Agree, but We expect to send a requests of such structure, let's stay with approach with 'totalRecords' for today. It works the same everywhere in data-import modules (mod-data-import, mod-source-record-storage, mod-source-record-manager)

Reply:
At this stage, I don't mind which route is chosen as long as the RAML is consistent with what the code expects (which wasn't the case, the code was expecting a raw JSON array)